### ISSUE TITLE
how to use in laravel 5.5 ?

### ISSUE THREAD

**Issue by aliworkshop** [koolkode/bpmn#4]
hi 

i want to use this plugin in laravel 5.5 but it can't be install for this
```php
 - Conclusion: remove symfony/console v3.4.2|install symfony/console v2.8.34
```

tnx alot

**Comment by razvancurcudel** [comment]
Did this fix work? I want to use it with Laravel as well.

**Comment by martinschroeder** [comment]
I released version 0.2.4 which should work with Symfony console v3 and v4. I hope this fixes the problem with laravel.

**Comment by razvancurcudel** [comment]
Thank you!
